pkgs/development/ocaml-modules/faillib/default.nix

```diff
@@ -0,0 +1,21 @@
+{stdenv, buildOcaml, fetchurl, herelib, camlp4}:
+
+buildOcaml rec {
+  minimumSupportedOcamlVersion = "4.02";
```

Comment:
Done. I was following the pattern I'd seen in the repo previously (plus, it had deps on 4.02-only herelib, so I couldn't test it with earlier than 4.02)